### PR TITLE
added home page

### PR DESCRIPTION
**Pull request by nikhiljugale007**
Added hero section,
On the home page added today's top deals, top category, and top brands section.
Design is responsive

Files to check
Home.jsx  => most code is of list used for storing dummy items
Home.css
adv_card.jsx => advertisement card
adv_card.css
discount_card.jsx 
discount_card.css

app.css no change files are appearing in changed section because of formatting changed

### CODE REVIEW

File: src/pages/homepage/Home.jsx [hunk 1]
```diff
@@ -1,5 +1,179 @@
...
+		{
+			brand_name: "hrx ",
+			image_src: hrxlogo,
+			image_alt: "hrx logo",
+		},
+		{
+			brand_name: "Wrogn ",
+			image_src: wrognlogo,
+			image_alt: "wrogn logo",
+		},
```
Comment (sri-rishi) [inline src/pages/homepage/Home.jsx]: Suggestion: these data files can me moved to another separate files like, `brandCategory.data.js `.

File: src/pages/homepage/Home.jsx [hunk 2]
```diff
@@ -1,5 +1,179 @@
...
+							affordable price. No cost EMI. Complete Protection. Exchange
+							offer.
+						</p>
+						<button className="btn btn-primary">Explore now</button>
+					</div>
+					<div className="img-section">
+						<img className="img-responsive" alt="hero" src={hero1} />
+					</div>
+				</div>
+			</div>
```
Comment (sri-rishi) [inline src/pages/homepage/Home.jsx:25]: Suggestion: You can make components for sections like `HeroSection components`

**Review by Naman-Saxena1 — APPROVED** [review]
Code looks fine 👍 
Just a small UI suggestion, you can make Explore Now button as cursor: pointer

**Review by sri-rishi — APPROVED** [review]
Great work, like the way how you are dynamically made home sections, just some suggestion which I mentioned are good to merge

**Comment by nikhiljugale007** [comment]
> Code looks fine 👍 Just a small UI suggestion, you can make Explore Now button as cursor: pointer

It is pointer already. Can you please check once.

File: src/pages/homepage/Home.jsx (hunk 2 above)
Comment (nikhiljugale007) [reply src/pages/homepage/Home.jsx:25]: I will try to further breakdown it once i complete all functionality. 
Thanks for suggestion.

File: src/pages/homepage/Home.jsx (hunk 1 above)
Comment (nikhiljugale007) [reply src/pages/homepage/Home.jsx]: moved data to new data files.

**Comment by nikhiljugale007** [comment]
> Code looks fine 👍 Just a small UI suggestion, you can make Explore Now button as cursor: pointer

It is already a pointer. Can you please check once?

**Comment by nikhiljugale007** [comment]
> Code looks fine 👍 Just a small UI suggestion, you can make Explore Now button as cursor: pointer

It is already pointer can you check once.

**Pull request merged**